INN/umbrella-citylimits
# 133 client qa updates

## Changes

This pull request makes the following changes:

- Adds new `Checked on Frontend?` field to CL Mailchimp ACF JSON that defaults to having newsletter be checked on the frontend https://github.com/INN/umbrella-citylimits/commit/28e13093c4a0b14b626774f0613f99ea6dd9f3f5
- Hides the hero image on author archive template https://github.com/INN/umbrella-citylimits/commit/c892f10398f80f4c742c201820c88a97ff67296c
- Updates `.and` to be lowercased https://github.com/INN/umbrella-citylimits/commit/c892f10398f80f4c742c201820c88a97ff67296c
- Reduces size of the top featured story title on the homepage https://github.com/INN/umbrella-citylimits/commit/7ce574c5e6cc9e2a9f81ba011afb7409f93d2762
- Copies `baseFontSize: 16px` from Largo in child theme `variables.less` and uses it to set the byline font size on the special projects seven posts widget so it's the same as on archive pages https://github.com/INN/umbrella-citylimits/commit/77e736044aa0dc52bb51c7183df8f30471ee5336
- https://github.com/INN/umbrella-citylimits/commit/74e55a1a24447d20efd072889f1533128d31f601 updates the `Share via email` mailto: link to:
```
Email Subject is: CityLimits.org: [HEADLINE]

Email Body is:
From City Limits:
[HEADLINE]
[DEK]
[Link]
``` 

## Why



For #133

## Testing/Questions

Features that this PR affects:

- Newsletter signup form
- Author archive template
- Bylines
- Homepage top featured story
- Share via email button


Questions that need to be answered before merging:


Steps to test this PR:

**Mapping the Future should be checked by default in the Newsletter signup:**
1. Make sure the new `Checked on Frontend?` field is appearing in the CL Mailchimp editor. If not, you'll need to add it in via the Custom Fields editor. Settings for it can be found here: https://github.com/INN/umbrella-citylimits/commit/28e13093c4a0b14b626774f0613f99ea6dd9f3f5
2. Make sure the checkboxes toggle checked/unchecked correctly based on their `Checked on Frontend?` field

**Remove lede image from author landing page template:**
1. Make sure the hero image is hidden on author archive pages

**Change "And" to "and" on article page main bylines:**
1. Find a post with multiple authors and verify the `and` is lowercased

**Reduce size of top headline on mobile homepage so its more in line with headlines for subsequent stories:**
1. View the homepage on mobile and verify the top featured story looks ok

**Update byline size on series landing pages so that it matches the sizes on category landing pages:**
1. Add the `City Limits Seven Series Posts` widget to a series page and make sure bylines are checked to be shown
2. Verify the byline font size is the same as the byline font size on category archives

**Change email share to use this format:**
1. Go to a post
2. Click the email share button
3. Verify all of the content is correct and formatting looks ok
